src/Command/CreatePolicyPlanCommand.php

```diff
@@ -222,9 +488,23 @@ public function askLayerGeometryName(
     ): ?string {
         $names = collect($this->connectionManager->getCachedGameSessionDbConnection($gameSessionId)
             ->executeQuery(
-                'SELECT JSON_EXTRACT(geometry_data, \'$.NAME\') FROM geometry WHERE geometry_layer_id = :layerId',
+                <<< 'SQL'
+                SELECT r.geometry_name
+                FROM (
+                    SELECT
+                      DISTINCT
+                      CONCAT(
+                        IFNULL(JSON_EXTRACT(geometry_data, '$.name'), ''),
+                        IFNULL(JSON_EXTRACT(geometry_data, '$.NAME'), ''),
+                        IFNULL(JSON_EXTRACT(geometry_data, '$.Name'), '')
```

Comment:
apparently different sea basins have an inconsistency in field name casing

Reply:
yes that's unfortunate - I guess we could lowercase all geometry data variable names upon session creation (as in: upon retrieval from GeoServer and storage in the session db), but I think that will create more problems...